src/libmongoc/tests/unified/runner.c

```diff
@@ -1291,6 +1339,21 @@ test_count_matching_events_for_client (
    return true;
 }
 
+static bool
+skip_cse_list_collections (const bson_t *event)
```

Comment:
```suggestion
// `is_keyvault_listcollections` returns true if a `listCollections` event produced by libmongoc should be ignored.
// The extra events are caused by operations on the key vault collection. Unlike other drivers, libmongoc does not
// create a separate client for key vault operations.
static bool
is_keyvault_listcollections (const bson_t *event)
```

Suggest commenting to describe motivation and renaming to clarify behavior.